### PR TITLE
Add Wordpress notifier service to the pub cluster

### PR DESCRIPTION
**Pull request by david-smith-ft**
http://git.svc.ft.com/projects/CP/repos/wordpress-notifier/browse

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -118,4 +118,9 @@ services:
   version: v1.2.0
 - name: vulcan.service
   version: v1.1.0
+- name: wordpress-notifier-sidekick@.service
+  count: 1
```
Comment (peteclark-ft) [inline services.yaml]: Count of 1 for the sidekick but 2 for the service?

File: wordpress-notifier@.service [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  --env "WORDPRESS_CONTENTAPI_KEY=$(etcdctl get /ft/_credentials/wordpress/content_api_key)" \
+  --env "VULCAN_HOST=%H:8080" \
+  --env "CMS_NOTIFIER_HOST=cms-notifier" \
+  up-registry.ft.com/coco/wordpress-notifier:$DOCKER_APP_VERSION'
+ExecStop=-/bin/bash -c 'docker stop -t 3 "$(docker ps -q --filter=name=%p-%i_)"'
+Restart=on-failure
+RestartSec=60
+
+[X-Fleet]
+Conflicts=%p@*.service
```
Comment (tamas-molnar) [inline wordpress-notifier@.service]: Should be wp-notifier

**Comment by tamas-molnar** [comment]
+1

**Comment by peteclark-ft** [comment]
👍

**Comment by kovacshuni** [comment]
+1

**Pull request merged**